### DISCUSSION TITLE
Sandpack Private Registry Sign-In

### DISCUSSION THREAD

**Discussion by rfenderdtn** [codesandbox/sandpack#937] [Q&A]
On [step 4 in the documentation](https://sandpack.codesandbox.io/docs/getting-started/private-packages), it says to sign in then you can see the preview. Does that apply to every visitor that encounters that Sandpack instance on the site?

**Comment by danilowoz** [comment]
Hey!

Yes, it does. Each user needs to sign-in in order to Sandpack be able to connect to the CodeSandbox server and retrieve private credentials or proxy the package request. If you're looking for something different, I'd like to suggest you take a look at this guide: [Self-host private packages](https://sandpack.codesandbox.io/docs/guides/hosting-the-bundler#self-host-private-packages)

**Reply by rfenderdtn** [reply]
With self-hosted private packages, would I still need a Team Pro account for my team?

**Reply by danilowoz** [reply]
No, it would not. You can have a free account, because it doesn't depend on the CodeSandbox server
